src/server/plugins/nunjucks/context.js

```diff
@@ -28,6 +28,16 @@ export function context(request) {
   }
 
   const { params, path } = request ?? {}
+
+  let cookieConsent
+  let cookieConsentUpdated
+
+  try {
+    cookieConsent = request?.state.cookieConsent ?? undefined
+    cookieConsentUpdated = request?.yar.flash('cookieConsentUpdated').at(0)
```

Comment:
Can we move this preference into the consent cookie too?

Then we can easily dismiss the cookie banner client-side without needing a page reload

I've suggested [something like `hide: false`](https://github.com/DEFRA/forms-runner/pull/571/files#r1875695628) and [updating the state to match](https://github.com/DEFRA/forms-runner/pull/571/files#r1876116756)

Reply:
Will do but in a follow-up PR, thanks for the feedback. For now I'll leave it.